roles/build-operand/files/image_builder.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: BuildConfig
+metadata:
+  labels:
+    app: ci-artifacts
+  name: "{{ build_operand_imagestream_name }}"
+  namespace: "{{ build_operand_namespace }}"
+spec:
+  output:
+    to:
+      kind: ImageStreamTag
+      name: "{{ build_operand_imagestream_name }}:{{ build_operand_tag }}"
+      namespace: "{{ build_operand_namespace }}"
+  resources: {}
+  source:
+    type: Git
+    git:
+      uri: "{{ build_operand_git_repo }}"
+      ref: "{{ build_operand_git_ref }}"
+    contextDir: "{{ build_operand_docker_context }}"
+  triggers:
+  - type: "ConfigChange"
+  strategy:
+    type: Docker
+    dockerStrategy:
+      buildArgs: {{ build_operand_docker_build_args }}
+      dockerfilePath: "{{ build_operand_dockerfile }}"
+      {{ build_operand_extra_docker_strategy }}
```

Comment:
This hacky yaml-sed-merge thingy is not appreciated by both the YAML and ansible lints (ansible-lint is running on this file, for some reason...). Would appreciate any tips on how to do this better

Reply:
can you use the `template` task instead of the `sed` hack?
from my 1st try in `gpu_operator_deploy_from_operatorhub` it works well

however the "template" file shouldn't change much, we'll have to study why it's complaining
it the linter is too strict for what we want, we can consider disabling some rules, it it's enforcing meaningless constraints or if we can't solve the warnings in a clean way